### ISSUE TITLE
fix: daw issues with the output

### ISSUE THREAD

**Issue by joyolababy** [hollance/TheKissOfShame#4]
Hiii. Some people have just reported that the plugin does not work on fl sutdio and ableton.
This is the link of the video (The issues are in the comments section): https://www.youtube.com/watch?v=cpYXG5zoO6A

![image](https://github.com/hollance/TheKissOfShame/assets/167874969/f724913c-cb84-4e7a-b9b0-850f51744479)

Sorry, for my english <3

**Comment by hollance** [comment]
Thanks for reporting. I didn't test on FL Studio or Ableton so it's very possible something's broken there.

**Comment by joyolababy** [comment]
> Thanks for reporting. I didn't test on FL Studio or Ableton so it's very possible something's broken there.

yw <3

**Comment by hollance** [comment]
I just tried on Ableton Live 11 on Mac and it works fine here. I'll try on Windows later since I don't have Ableton installed there at the moment.

**Comment by grxves** [comment]
i just tried to use the effect on FL Studio on Windows and sadly that doesn't work.

**Comment by hollance** [comment]
OK so I installed the trial of FL Studio on Windows 10 and the plug-in loads just fine but it outputs no audio. Is that also what you were seeing @grxves?

**Comment by grxves** [comment]
yes exactly it has all the animations and so on but no output

**Comment by hollance** [comment]
OK, found the bug and fixed it. There are a few other fixes I want to include in the next release, so it will have to wait for that.

**Comment by joyolababy** [comment]
<3

**Comment by hollance** [comment]
The FL Studio issue should be fixed by release 1.0.1, which is out now. Not sure if this also fixes the Ableton issue (which I could not reproduce). Let me know if it works!

**Comment by grxves** [comment]
works now! just one (actually 2) question since i didn't find a user manual: what do enviroment and the cross knob do?

**Comment by hollance** [comment]
Good to hear! The README kind of explains what these knobs do. The cross knob (Shame) will add wow and flutter. The environment only works if you set it to Hurricane Sandy and dial up the Age button. It adds noise and scratches to the sound.

**Comment by grxves** [comment]
I see.
I was about to check the read me but it wasn't in the zip file so i thought it would be about compiling.
I was wondering what the shame knob did cus it was so much happening at the same time.
Really cool sound to everything btw

**Comment by hollance** [comment]
Yeah I was thinking about doing a nice manual that goes in the zip file but haven't done it yet.

**Comment by RitheshKumar** [comment]
Speaking of manuals, @hollance would you have a signal chain graph for the DSP? I'm inspecting the code, perhaps I could create one if it doesn't exist.

**Comment by hollance** [comment]
@RitheshKumar I did a full write-up on my blog: https://audiodev.blog/kiss-of-shame/

**Comment by RitheshKumar** [comment]
Fantastic! :)
